02-inspect-file.md

```diff
@@ -169,7 +169,7 @@ Instead of simply counting the files that match the search criteria, save them t
 This is done with the redirection operator, `>`.
 
 ~~~ {.bash}
-gunzip -c counts-raw.txt.gz | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
+zcat counts-raw.txt.gz | cut -f11 | grep "Evolutionary Biology" | grep "Cell Biology" > evo-cell-bio.txt
```

Comment:
Ah, ok, I see what you're doing here now. This all makes sense. I can quickly change that back and reissue the PR if that's easiest?